tests/ops/op_math/test_decompositions.py

```diff
@@ -18,18 +18,15 @@
 
 from functools import reduce
 
+import numpy as np
 import pytest
 from gate_data import CNOT, SWAP, H, I, S, T, X, Y, Z
 from scipy import sparse
 
 import pennylane as qml
```

Comment:
can we `from pennylane import math` here and replace all the `qml.math` as `math` here?

Reply:
This is in the test suite (as opposed to in the source code) so I think it's fine. The motivation for importing submodules instead of directly using `import pennylane as qml` in source code is to avoid circular dependencies between different layers of the software architecture. When we're using the package from the outside (i.e., the test suite), it's perfectly fine to access things from the top level.